fleetspeak/server-wheel/setup.py

```diff
@@ -14,6 +14,8 @@ def finalize_options(self):
 
   def get_tag(self):
     impl, abi_tag, plat_name = bdist_wheel.get_tag(self)
+    if plat_name == "linux_x86_64":
```

Comment:
Please add a comment re why this is needed.

Reply:
Done.